resources/definitions/fdmprinter.def.json

```diff
@@ -8125,6 +8125,14 @@
                     "resolve": "max(extruderValues('raft_base_wall_count'))",
                     "settable_per_mesh": false,
                     "settable_per_extruder": false
+                },
+                "group_outer_walls":
+                {
+                    "label": "Group Outer Walls",
+                    "description": "When enabled outer walls of different islands are printed in-sequence. When enabled the this limits the amount of flow changes, while disabling reduces the amount of travels",
```

Comment:
Outer walls of different islands in the same layer are printed in sequence. When enabled the amount of flow changes is limited because walls are printed one type at a time, when disabled the number of travels between islands is reduced because walls in the same islands are grouped